packages/jasmine-parameterized/CONTRIBUTING.md

```diff
@@ -1,9 +1,11 @@
-# Contributing to Jasmine Parameterized
+# Contributing
```

Comment:
Looks like this CONTRIBUTING may have originally been modelled after the one from `xliff-to-json-converter`. You could consider making similar changes there. But that package is touched so infrequently that I don't mind if you leave it as-is.

Reply:
Won't do it as part of this PR. Think it'll be on my back of mind tech debt plate to give the treatment to other READMEs. Angular-workspace just renders as a blank one lol